### ISSUE TITLE
Mix docs silent on weight endpoints

### ISSUE THREAD

**Issue by pnorman** [less/less-docs#409]
The [mix docs](http://lesscss.org/functions/#color-operations-mix) are silent on if a weight of 0 is 100% of the first colour or 100% of the second colour. Also, no examples are other than 50%.

**Comment by seven-phases-max** [comment]
Yep, that's missing (usually, when developed by descendants of Greek/Roman civilizations, the assumed default direction is left-to-right/clock-wise, thus 0% -> full left/first/a color, 100% -> full right/second/b one).
